docroot/profiles/custom/cgov_site/modules/custom/cgov_core/config/install/user.role.advanced_editor.yml

```diff
@@ -5,12 +5,3 @@ id: advanced_editor
 label: 'Advanced Editor'
 weight: -6
 is_admin: null
-permissions:
```

Comment:
Since these are only partial permissions, removed from config and placed in hook_install().